bindings.c

```diff
@@ -4065,8 +4076,8 @@ int proc_open(const char *path, struct fuse_file_info *fi)
 int proc_access(const char *path, int mask)
 {
 	/* these are all read-only */
-	if ((mask & ~R_OK) != 0)
-		return -EACCES;
+	if (access(path, mask) < 0)
```

Comment:
Right, totally missed the read-only line above. Users should still be able to access `/var/lib/lxcfs/proc`. I think I'll just do:

```
if (strcmp(path, "/proc") == 0)
        return 0;
```

and leave the original check unchanged.